apps/server/src/modules/learnroom/service/learnroom-config.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { LearnroomConfig } from '../learnroom.config';
+
+@Injectable()
+export class LearnroomConfigService {

```

Comment:
This service should be removed. Please not add default values at this place it should be added to default.schema.json with production default, if you want to have different value in test then mock it and if you want to have different value for local you can add it to the additional config file. But i expect it should be the same like in production.

The config service is 
private readonly configService: ConfigService<LearnroomConfig, true>
and can directly used. But please avoid addint the result auf the config to a class property.